src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+
+import java.util.List;
+
+
+/**
+ * Lists all persons in the address book to the user in alphabetical order.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Lists all persons in the address book to the user in alphabetical order.\n"
+            + "Example: " + COMMAND_WORD;
+
```

Comment:
Try to cut down on extra unnecessary lines if possible. Although it might be easier to read at first, it will lengthen the number of lines which results in scrolling through more code.

The best compromise would be to group related lines together and then leave a space for the next chunk of logically related code :)